src/atoms/ArrowBullet.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import styled from 'styled-components';
+
+import { FontIcon, getColor } from '..';
+
+const Bullet = styled.a`
+    color: ${getColor('type')};
+    font-size: ${props => props.theme.variables.headingMediumSize};
+    line-height: ${props => props.theme.variables.headingMediumLineHeight}
+    font-family: ${props => props.theme.variables.mainFont};
+    transition: all 300ms cubic-bezier(0.165, 0.84, 0.44, 1);
+    display: inline-block;
+    cursor: pointer;
+
+    &:hover {
+        color: ${getColor('primary')};
+        transform: translateX(1em);
```

Comment:
Consider variables.horizontalBase instead of hardcoding a magic width

Reply:
Done